src/consensus/params.h

```diff
@@ -23,7 +23,8 @@ enum DeploymentPos {
     DEPLOYMENT_REALLOC, // Deployment of Block Reward Reallocation
     DEPLOYMENT_DIP0020, // Deployment of DIP0020, DIP0021 and LMQ_100_67 quorums
     DEPLOYMENT_DIP0024, // Deployment of DIP0024 (Quorum Rotation) and decreased governance proposal fee
-    DEPLOYMENT_V19,     // Deployment of Basic BLS, AssetLocks, EHF
+    DEPLOYMENT_V19,     // Deployment of Basic BLS, AssetLocks
```

Comment:
AssetLocks also goes to V20

Reply:
I will open a separate chore PR to fix the comments. They are wrong in both v19 and v20 descriptions.